src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/ItemSettings.js

```diff
@@ -53,6 +53,7 @@ export const ItemSettings = memo(
                 path={web.path}
                 onChange={onChange}
                 currentItemLangID={props.item.meta.langID}
+                dispatch={props.dispatch}
```

Comment:
This seems unrelated to the error message on the sentry bug referenced; 

> Cannot read properties of undefined (reading 'data')

That appears to be that the response is empty but we are still attempting to read the `data` property

Reply:
Added Optional Chaining to `data`, but leaving the props.dispatch as ItemParent is is missing dispatch.